lib/hanami/assets/configuration.rb

```diff
@@ -524,7 +567,12 @@ def load!
       # @api private
       def compile_path(source)
         result = prefix.join(source)
-        result = digest_manifest.resolve(result) if digest
+
+        if digest
+          result = digest_manifest.resolve(result)
```

Comment:
@accuser We're leaking internal informations of `DigestManitfest`. We should think to expose a new method `#target`. So we can do:

```ruby
result = digest_manifest.target(result) if digest
```

It's implementation should be

```ruby
def target(path)
  resolve(result).fetch('target')
end
```